src/Pipfile

```diff
@@ -23,6 +23,7 @@ requests = "*"
 django-fsm = "*"
 django-phonenumber-field = {extras = ["phonenumberslite"], version = "*"}
 boto3 = "*"
+typing-extensions ='*'
```

Comment:
I was running into an [error where this was not available in CI](https://github.com/cisagov/getgov/actions/runs/5146963106/jobs/9266729233?pr=667). Not sure why it is now explicitly required

Any insight appreciated 😄